google-chart.html

```diff
@@ -328,6 +328,26 @@
         value: null,
         observer: 'drawChart'
       },
+
+      /**
+       * The Image URI for the currently displayed graph.
+       *
+       * This value can be used as the `src` attribute of an `img` tag.
+       *
+       * Example:
+       * <pre>
```

Comment:
I was just copying from the other examples.
I can fix them all in a single clean up commit.